pkg/testcase/secretsencrypt.go

```diff
@@ -106,28 +105,29 @@ func verifyStatusOutput(action, stdout string) {
 }
 
 func getNodeIps(etcdNodes, cpNodes []shared.Node) []string {
-	var sarray []string
+	var nodeIpsArr []string
```

Comment:
sorry think i commented here but seems it doesnt

lets naming better accordingly to use case x language idiomacy 
nodeIPs , its great 

no need to specify the type on the name, execpt in more complex flow or logic 
also in golang this is not an array, its a slice --  to declare an array you need to add the lenght

Reply:
for sure, it serves for any other places that u might have this also

Reply:
fixed